File: PASCAL/Micro01.pas
program micro01 ;
Var cel , fahr : real;
begin
    writeln('Tabela de conversao Celsius -> Fahrenheit ');
    write('Digite a temperatura em Celsius : ');
    readln(cel);
    fahr := (9 * cel + 160 ) / 5;
    write('A nova temperatura e: ');
    write(fahr) ;
    writeln(' F ') ;
end.

File: IR LLVM/Micro01/Micro01.c
#include <stdio.h>
#include <stdlib.h>

int main(){
    double cel;
    double fahr;

    printf("Tabela de conversão Celsius -> Fahrenheit\n");
    printf("Digite a temperatura em Celsius: ");
    scanf("%lf", &cel);
    fahr = ((9 * cel ) + 160) / 5;
    printf(" A nova temperatura é : %lf F\n", fahr);
    return 0;
    
}